doc/user_guide/concurrent_substep.html

```diff
@@ -15494,7 +15482,7 @@ <h4>Substeps with statements after <code>sos_run</code> are not executed in para
 <script>
 function editme() {
     let filepath = window.location.pathname.slice(0, -5).replace('doc/', 'src/');
-    let new_url = `http://128.135.144.117:8000/hub/user-redirect/notebooks/${filepath}.ipynb`;
+    let new_url = `http://sosworkflows.com:8000/hub/user-redirect/notebooks/${filepath}.ipynb`;
```

Comment:
Is this what you meant? @BoPeng

Reply:
sosworkfliws.com serves on regular port, not on 8080.

Reply:
sosworkfliws.com serves on regular port, not on 8080.

Reply:
sosworkfliws.com serves on regular port, not on 8080.

Reply:
Does regular port mean the ":8000" should not be in the url?

Reply:
Yes, you should have tested that URL before using it.